lesson_9/cg_ed_1.md

```diff
@@ -0,0 +1,118 @@
+---
+title: CG Education - JS 1
+permalink: /cg_ed_1
+---
+
+# CG Education - JavaScript Part I
+
+**Plan:** List steps/tasks in [Trello Boards](https://trello.com/cg_webdev_ss_2018)
+
+**Do:**  Download app to your phone and play (instructions below).
+
+**Record:** [Lesson 9.3 CoderGirl Education Project Part 1](https://learn.launchcode.org/courses/131/assignments/7223)
```

Comment:
@ktmathews89 please update with the real link, I forgot!

Reply:
Created a new lesson on Canvas for lesson 9